src/main/java/DianaAssistant.java

```diff
@@ -0,0 +1,155 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class DianaAssistant {
+    private static final String ENCLOSURE = "------------------------------";
+
+    public void interact() {
+        printWelcomeMessage();
+        Scanner scanner = new Scanner(System.in);
+        List<Task> list = new ArrayList<>();
+
+        String input;
+        while (true) {
+            input = scanner.nextLine();
```

Comment:
Could you move the declaration of input into the while loop (to line 15), and put it together with the instantiation (taking in of scanner input)?